src/main/java/com/contrastsecurity/http/SecurityCheckForm.java

```diff
@@ -52,12 +49,31 @@
     @SerializedName("origin")
     private String origin = "OTHER";
 
-    public SecurityCheckForm(String applicationId) {
+    /**
+     * The job start time of the jenkins job
+     * @param jobStartTime New value for job start time.
+     * @return the start time of the jenkins job
+     */
+    @SerializedName("job_start_time")
+    private Long jobStartTime;
+
+    public static SecurityCheckForm securityCheckForm(String applicationId) {
```

Comment:
why are these constructors are static?

Reply:
If you want to make these static factories instead, we should depricate the old constructors and add 

```
public static SecurityCheckForm newSecurityCheckFormWithId(String applicationId);
public static SecurityCheckForm newSecurityCheckFormWithNameAndLanguage(String applicationName, AgentType agentLanguage);
```
there is probably a better naming scheme...

Reply:
lol, I read it wrong, these are constructors.